README.md

```diff
@@ -77,7 +77,7 @@ fn main() {
 
     let sample_text=r#"
     {
-        # specify rate in requests/second
+        // specify rate in requests/second
```

Comment:
What is the purpose in changing this comment type?

Reply:
Oh, so the markdown test generator turns code blocks into doctests, and I was really confused about why there was a parse error in hjson until I realized that `#` is a special character in doctests, and apparently even in string literals! Like for:
```rust
/// Example
/// ```
/// # use foo::Bar;
/// Bar::baz();
/// ```
struct Bar;
```
The `use` line would get compiled in the doctests, but it would be hidden in the generated documentation. So basically, rustdoc was recognizing the `#` as a special character and just removing it, which would lead to the string literal missing the `#` and hjson failing to parse a line of just `specify rate in requests/second`